src/Robo/Commands/Tests/TestsCommandBase.php

```diff
@@ -157,7 +157,7 @@ protected function findChrome() {
    */
   protected function checkChromeVersion($bin) {
     $version = (int) $this->getContainer()->get('executor')
-      ->execute("'$bin' --version | cut -f1 -d'.' | rev | cut -f1 -d' ' | rev")
+      ->execute("'$bin'", "--version | cut", "-f1", "-d'.' | rev | cut -f1 -d'\" ' | rev",)
```

Comment:
Why does this make a difference? I saw the error but i was unsure how to test for it or what could have change.

Reply:
never-mind, its due to the command arrays change ah. That's interesting.